src/popup/content.jsx

```diff
@@ -26,6 +27,23 @@ class App extends Component {
     setTimeout(() => {
       this.getContent()
     }, 1)
+    key.filter = () => true
+    key('space', () => {
+      document.querySelector("body > div > div._container > div > div").scrollBy(0, 150)
```

Comment:
How about using React refs rather than DOM APIs and adding more animation effects?
```suggestion
      this.pageRef.current.scrollBy({ top: 150, left: 0, behavior: 'smooth' })
```